### DISCUSSION TITLE
Markdown content field ignoring new lines

### DISCUSSION THREAD

**Discussion by nicktc** [statamic/cms#4806] [Q&A]
Hi,

I have a collection with a 'markdown' content field. I enabled the 'Automatic Line Breaks'. Yet most of the line breaks are ignored:

<img width="1792" alt="Schermafbeelding 2021-11-23 om 17 12 16" src="https://user-images.githubusercontent.com/2048475/143061454-5e6a7883-8398-47e8-a16d-b6b110a530cb.png">

See also the field setting:

<img width="1559" alt="Schermafbeelding 2021-11-23 om 17 14 38" src="https://user-images.githubusercontent.com/2048475/143061739-77c04a82-85ad-4a6d-b35d-befdeda2c72c.png">

Am I doing something wrong?

**Comment by jasonvarga** [comment]
I believe it's working as intended.

You can't spam line breaks like that. It's just going to collapse it into two paragraphs.

```
A paragraph







Another paragraph
```
```html
<p>A paragraph</p>
<p>Another paragraph</p>
```

What that automatic line break setting does is allow you to press enter in the middle of a paragraph and it'll create a `<br>`.

```
My paragraph[no space]
new line
```
outputs
```html
<p>My paragraph<br>new line</p>
```


With it disabled, you have to use the standard markdown behavior of ending a line with two spaces in order to get a `<br>`.


```
My paragraph[no space]
new line

My paragraph[space][space]
new line
```
```html
<p>My paragraph new line</p>
<p>My paragraph<br>new line</p>
```

**Comment by nicktc** [comment]
I think I know the problem, and it is related to my other ticket. A new line or even two space's do create a `<p>`-tag, but Tailwind is making a paragraph nog behave as a paragraph :-(. So although technically it was creating these p-tags, the browser didn't show them. Thats a massive downside of Tailwind.

**Reply by jasonvarga** [reply]
Use the typography plugin to style them.

**Reply by nicktc** [reply]
Yeah, haven't worked with that plugin, but will definitely give it a try! Thx!

**Comment by joshuavanderpoll** [comment]
Has there been any implementation for this without the use of another plugin already?
